### ISSUE TITLE
math.div function is undefined

### ISSUE THREAD

**Issue by eseselpg** [ionic-team/stencil-sass#62]
A project using the latest stable version of `material-components-web` component fails to build due to missing `math.div` SASS function.

Could we have `sass` dependency updated?
```
[ ERROR ]  sass error: node_modules/@material/theme/_theme-color.scss:33:30
           Undefined function.

     L32:  @function _linear-channel-value($channel-value) {
     L33:    $normalized-channel-value: math.div($channel-value, 255);
     L34:    @if $normalized-channel-value < 0.03928 {
```

**Comment by jan-paulus** [comment]
Will be fixed once https://github.com/ionic-team/stencil-sass/pull/55 is merged. But the stencil team seems to be busy with other things as the moment.

**Comment by splitinfinities** [comment]
Hey there, thank you for the patience getting back to you. The new team is getting started and we're working through the backlog now. I'm going to take a peek at some of the solutions here around peer deps and updating the internal version and circle back soon. 

I think updating to be a peer dep would likely be a breaking change for this package, but I'm not against it! BRB soon with more.

**Comment by klh** [comment]
> Will be fixed once #55 is merged. But the stencil team seems to be busy with other things as the moment.

> Hey there, thank you for the patience getting back to you. The new team is getting started and we're working through the backlog now. I'm going to take a peek at some of the solutions here around peer deps and updating the internal version and circle back soon.
> 
> I think updating to be a peer dep would likely be a breaking change for this package, but I'm not against it! BRB soon with more.

could you sorta tell us whats going on ? - the whole stack is growing dusty - and PRs not getting picked up across the board, so people are actually starting to look towards open-cw and lit.dev instead due to the very slow update / progress (months without any reactions)

**Comment by splitinfinities** [comment]
Yeah @klh! We're getting all of our ducks in a row now that the new team has been working on the core repo for a while so that we can more reliably support everything else within the `@stencil` npm scope. If you join us here https://youtu.be/47h9AVlTaF0 tomorrow I can answer any questions you may have! It's been my priority to organize all the repos so we can effectively work on them.
